draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -841,6 +867,26 @@ the SD-JWT.
 
 # Privacy Considerations {#privacy_considerations}
 
+## Confidentiality during Transport
+
+If the SD-JWT and associated Disclosures are transmitted over an insecure
+channel during issuance or presentation, an adversary may be able to
+intercept and read the End-User's personal data or correlate the information with previous uses of the same SD-JWT.
```

Comment:
```suggestion
intercept and read the End-User's personal data.
```
How does the insecure channel related to correlating the multiple usage of the same SD-JWT..? isn't it more of an identifier/signing key not being pairwise issue..?

Reply:
The solution to both problems is the same, but 

1. colluding verifiers identifying a person using the same signing keys, and
2. attackers observing the wire and seeing the same credential

are two different problems.